Build fails on SVG error

Attempting to build with command ``PATH_TO_XMLLINT="/usr/bin/xmllint" PATH_TO_JING="/usr/bin/jing" PATH_TO_SAXON_JAR="../saxon9he.jar" PATH_TO_TEI_STYLESHEETS="../Stylesheets" ./build.sh build`` gives this error: 
[mei-err.log](https://github.com/music-encoding/music-encoding/files/3008993/mei-err.log)

Hi, 
the essential message in your error log is this line:
`Caused by: org.xml.sax.SAXParseException; systemId: http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/svg11.rng; lineNumber: 6; columnNumber: 3; The element type "hr" must be terminated by the matching end-tag "</hr>"`

To my opinion this is a bug in the TEIC/Stylesheets project as it does not resolve correctly the mentionned URL. Instead you get an 301 Error msg from the ngix server where the `hr` Error is detected.

We can solve this also by setting the URL to `https://www.tei-c.org/release/xml/tei/custom/schema/relaxng/svg11.rng` in the MEI sources or just in your odd file.
HTH, franz

--- a/customizations/mei-CMN.xml
+++ b/customizations/mei-CMN.xml
@@ -92,7 +92,7 @@
         <moduleRef key="MEI.visual"/>
 
         <!-- Include SVG -->
-        <moduleRef url="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/svg11.rng"
+        <moduleRef url="https://www.tei-c.org/release/xml/tei/custom/schema/relaxng/svg11.rng"
           prefix="svg_">
           <content>
             <rng:define name="mei_model.graphicLike" combine="choice">

--- a/customizations/mei-Mensural.xml
+++ b/customizations/mei-Mensural.xml
@@ -92,7 +92,7 @@
         <moduleRef key="MEI.visual"/>
 
         <!-- Include SVG -->
-        <moduleRef url="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/svg11.rng"
+        <moduleRef url="https://www.tei-c.org/release/xml/tei/custom/schema/relaxng/svg11.rng"
           prefix="svg_">
           <content>
             <rng:define name="mei_model.graphicLike" combine="choice">

--- a/customizations/mei-Neumes.xml
+++ b/customizations/mei-Neumes.xml
@@ -92,7 +92,7 @@
         <moduleRef key="MEI.visual"/>
 
         <!-- Include SVG -->
-        <moduleRef url="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/svg11.rng"
+        <moduleRef url="https://www.tei-c.org/release/xml/tei/custom/schema/relaxng/svg11.rng"
           prefix="svg_">
           <content>
             <rng:define name="mei_model.graphicLike" combine="choice">

--- a/customizations/mei-all.xml
+++ b/customizations/mei-all.xml
@@ -97,7 +97,7 @@
         <moduleRef key="MEI.visual"/>
 
         <!-- Include SVG -->
-        <moduleRef url="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/svg11.rng"
+        <moduleRef url="https://www.tei-c.org/release/xml/tei/custom/schema/relaxng/svg11.rng"
           prefix="svg_">
           <content>
             <rng:define name="mei_model.graphicLike" combine="choice">

--- a/customizations/mei-all_anyStart.xml
+++ b/customizations/mei-all_anyStart.xml
@@ -133,7 +133,7 @@
         <moduleRef key="MEI.visual"/>
 
         <!-- Include SVG -->
-        <moduleRef url="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/svg11.rng"
+        <moduleRef url="https://www.tei-c.org/release/xml/tei/custom/schema/relaxng/svg11.rng"
           prefix="svg_">
           <content>
             <rng:define name="mei_model.graphicLike" combine="choice">

